Dockerfile

```diff
@@ -1,10 +1,7 @@
-FROM centos:8
-
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && \
-    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+FROM amazonlinux:2
 
 RUN yum update -y && \
-    yum module install nodejs:14/development -y && \
+    curl --silent --location https://rpm.nodesource.com/setup_14.x  | bash - && \
```

Comment:
The setup script installs a local repository configuration. As a result, it would be beneficial to have this executed before the `yum update -y && \` line. Best to have it on its own line as it's not directly related to the `yum` command. Similar to the way the v10 dockerfile is configured.

E.g.:
```
RUN curl --silent --location https://rpm.nodesource.com/setup_14.x | bash -

RUN yum update -y && \
```

Also, the `nss` package is installed by default in Amazon Linux 2 so can be omitted from the lsit of packages.